webknossos/webknossos/dataset/_utils/pims_images.py

```diff
@@ -178,10 +200,23 @@ def __init__(
                     else:
                         self._img_dims = "yxc"
                     self._iter_dim = "z"
+                elif len(images.shape) == 5:
+                    if images.shape[2] == 1 or (
+                        images.shape[2] == 3 and images.dtype == np.dtype("uint8")
+                    ):
+                        self._img_dims = "cyx"
+                    else:
+                        self._img_dims = "yxc"
+                    self._iter_dim = "z"
+                    self._timepoint = 0
+                    if images.shape[0] > 1:
+                        self._possible_layers["timepoint"] = list(
+                            range(0, images.shape[0])
+                        )
```

Comment:
some comments would be nice. jumping from another domain into this code is quite hard :see_no_evil: